modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccelerators.testcase

```diff
@@ -44,6 +44,37 @@ definition {
 		}
 	}
 
+	test AssertPromoProductPriceTotalIsDifferent {
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "CommerceAccelerators#AssertPromoProductPriceTotalIsDifferent";
+
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		Navigator.openWithAppendToBaseURL(urlAppend = "web/minium");
+
+		CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Account Management");
+
+		CommerceAcceleratorsInitializer.createNewAccountViaFront(
+			accountName = "Commerce Account",
+			adminEmailAddress = "test@liferay.com");
+
+		Navigator.openWithAppendToBaseURL(urlAppend = "web/minium");
+
+		Pagination.changePagination(itemsPerPage = "60");
+
+		FrontStore.addInitializerProductToCart(
+			productName = "Brake Pads",
+			productQuantity = "1");
+
+		CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Pending Orders");
+
+		Click(locator1 = "//a[contains(text(),'View')]");
+
+		AssertTextEquals(
+			locator1 = "CommerceAccelerators#ORDER_DETAILS_PRICE",
+			value1 = "$ 18.90");
+	}
+
```

Comment:
@nikki-pru this test is not part of any test plan. We are reviewing the steps so at the moment it can be removed. Thanks

Reply:
In progress

Reply:
Done